nautobot/dcim/views.py

```diff
@@ -2684,15 +2621,13 @@ class CableBulkImportView(generic.BulkImportView):
 
 
 class CableBulkEditView(generic.BulkEditView):
-    # v2 TODO(jathan): Replace prefetch_related with select_related
     queryset = Cable.objects.prefetch_related("termination_a", "termination_b")
```

Comment:
For my understanding - no select_related here because these are generic foreign keys?

Reply:
Yep, the django docs are a little confusing on this point IMO, but it is here https://docs.djangoproject.com/en/dev/ref/models/querysets/#django.db.models.query.QuerySet.prefetch_related